src/mysql_dirver/db.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
We will have many drivers. Instead of making modules like "xxx_driver", I would make a module "drivers/" i there we will have many drivers "drivers/mysql", "drivers/postgresql", etc.